src/components/scene/auto-enter-vr.js

```diff
@@ -2,9 +2,9 @@ var registerComponent = require('../../core/component').registerComponent;
 var utils = require('../../utils');
 
 /**
- * Automatically enter VR, either upon vrdisplayactivate (e.g. putting on Rift headset)
+ * Automatically enter VR either upon `vrdisplayactivate` (e.g. putting on Rift headset)
  * or immediately (if possible) if display name contains data string.
- * The default data string is 'GearVR' for Carmel browser which only does VR.
+ * Default data string is `GearVR` for Carmel browser which only does VR.
  */
 module.exports.Component = registerComponent('auto-enter-vr', {
```

Comment:
See #2707 - now that Oculus Browser is widely available, I think that auto-enter-vr should not be added to scene by default anymore - if developing for Carmel, easy enough to add back in